applications/GeoMechanicsApplication/custom_elements/geo_curved_beam_element.hpp

```diff
@@ -85,8 +84,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoCurvedBeamElement
         mThisIntegrationMethod = this->GetIntegrationMethod();
     }
 
-    /// Destructor
-    virtual ~GeoCurvedBeamElement() {}
+    virtual ~GeoCurvedBeamElement() = default;
```

Comment:
I believe we can remove the keyword `virtual` here, since at the very top of the class hierarchy, the destructor has already been declared `virtual`.

Reply:
Done